include/sys/mntent.h

```diff
@@ -71,8 +73,16 @@
 #define	MNTOPT_STRICTATIME "strictatime" /* strict access time updates */
 #define	MNTOPT_NOSTRICTATIME "nostrictatime" /* No strict access time updates */
 #define	MNTOPT_LAZYTIME "lazytime"	/* Defer access time writing */
+#ifdef __linux__
 #define	MNTOPT_SETUID	"suid"		/* Both setuid and devices allowed */
 #define	MNTOPT_NOSETUID	"nosuid"	/* Neither setuid nor devices allowed */
+#elif defined(__FreeBSD__)
+#define	MNTOPT_NOSUID	"nosuid"	/* Neither setuid nor devices allowed */
```

Comment:
Is there a need to define `MNTOPT_NOSUID` at all since it's only used from `cmd/mount_zfs/mount_zfs.c` which is only built for Linux.

Reply:
I guess not.